Modelica_LinearSystems2/WorkInProgress/Controller/KalmanFilter/SRKF_inner2.mo

```diff
@@ -2,6 +2,7 @@ within Modelica_LinearSystems2.WorkInProgress.Controller.KalmanFilter;
 model SRKF_inner2 "Discrete State Space block"
 
   import Modelica;
+  import MSLcholesky = Modelica.Math.Matrices.cholesky;

```

Comment:
importing a function with a different name looks weird. I would either import it as just "cholesky" or import MatricesMSL as in other places and use MatricesMSL.cholesky

Reply:
Done.